bearer/bearer.go

```diff
@@ -43,8 +43,13 @@ func (b *Generator) NewBearer(email string, currentEpoch uint64, msPerEpoch int6
 	epochs := uint64(b.config.ObjectMaxLifetime.Milliseconds() / msPerEpoch)
 
 	t := eacl.CreateTable(b.config.ContainerID)
+	rec := eacl.CreateRecord(eacl.ActionDeny, eacl.OperationPut)
+	rec.AddFilter(eacl.HeaderFromObject, eacl.MatchNotPresent, object.AttributeContentType, "")
+	t.AddRecord(rec)
+	eacl.AddFormedTarget(rec, eacl.RoleOthers)
```

Comment:
Better have a new record.

Reply:
Refactored in the separate commit, to be able to easily manage order and rules in the future